src/QirRuntime/samples/StandaloneInputReference/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+#=======================================================================================================================
+# This is a reference make file on how to build a QIR-based standalone executable.
+#=======================================================================================================================
+
+add_executable(qir-input-reference-standalone
+  qir-driver.cpp
+)
+
+# This is where the generated QIR file is specified.
+compile_from_qir(qir-standalone-input-reference qir_standalone_input_reference_target)
+
+target_link_libraries(qir-input-reference-standalone PUBLIC
+  ${QIR_UTILITY_LIB} # created by compile_from_qir
+  ${QIR_BRIDGE_UTILITY_LIB}
+  ${QIR_BRIDGE_QIS_UTILITY_LIB}
+  qir-rt-support
+  qir-qis-support
+  simulators
+)
+
+target_include_directories(qir-input-reference-standalone PUBLIC
+  "${standalone_includes}"
+  "${public_includes}"
+  "${PROJECT_SOURCE_DIR}/lib/QIR"
+)
+
+add_dependencies(qir-input-reference-standalone qir_standalone_input_reference_target)
+
+install(TARGETS qir-input-reference-standalone RUNTIME DESTINATION "${CMAKE_BINARY_DIR}/bin")
```

Comment:
We should probably have some tests for the sample, even if not using catch2. Something that at least runs the built binary...
```
    add_test(NAME ${target} COMMAND ${target}
    )
```
But I don't know what kind of output it should produce to be visible in CI...

Reply:
Added a test for the sample in the cmake file and also in the test.py file.